Author: Jelmer Vernooij <jelmer@openchange.org>
Description: Link mapiproxy against -ldl to prevent missing symbol errors
Status: cherry-picked from upstream

=== modified file 'Makefile'
--- old/Makefile	2012-03-19 13:23:38 +0000
+++ new/Makefile	2012-03-19 13:40:08 +0000
@@ -771,7 +771,7 @@
 							mapiproxy/libmapiproxy/modules.po			\
 							libmapi.$(SHLIBEXT).$(PACKAGE_VERSION)
 	@echo "Linking $@"
-	@$(CC) -o $@ $(DSOOPT) $(LDFLAGS) -Wl,-soname,libmapiproxy.$(SHLIBEXT).$(LIBMAPIPROXY_SO_VERSION) $^ -L. $(LIBS) $(TDB_LIBS)
+	@$(CC) -o $@ $(DSOOPT) $(LDFLAGS) -Wl,-soname,libmapiproxy.$(SHLIBEXT).$(LIBMAPIPROXY_SO_VERSION) $^ -L. $(LIBS) $(TDB_LIBS) $(DL_LIBS)
 
 mapiproxy/libmapiproxy.$(SHLIBEXT).$(LIBMAPIPROXY_SO_VERSION): libmapiproxy.$(SHLIBEXT).$(PACKAGE_VERSION)
 	ln -fs $< $@

